### ISSUE TITLE
Baseline (for big old projects)

### ISSUE THREAD

**Issue by aleien** [shyiko/ktlint#220]
Hello (:

Is it possible to generate baseline file (list all current issues as ignored and only perform check on new ones)? Or should we use script to get list of changes and run ktlint on them using "args"?
Also is there any limit on how many paths could be listed in "args"?

Thank you in advance.

**Comment by shyiko** [comment]
> Is it possible to generate baseline file (list all current issues as ignored and only perform check on new ones)?

no

> Or should we use script to get list of changes and run ktlint on them using "args"? Also is there any limit on how many paths could be listed in "args"?
```sh
cd project-dir/
ktlint -F # auto-correct
```

**Comment by aleien** [comment]
Thanks for fast response.
Yeah, I understand that you can autocorrect everything at once (: But if you have big project with 10 developers working on it at once it usually will be difficult to merge these changes. 

If I write script that will list me all file paths that were changed in one pull-request, can I provide these paths to ktlint? Is there any limit for how much explicit paths can I give ktlint to analyze?

**Comment by shyiko** [comment]
Sure. To my knowledge, there is no limit other than OS-specific "max length of args" (e.g. `getconf ARG_MAX` on \*nix) (additional shell-specific restrictions might apply).

**Comment by amrfarid140** [comment]
Looks like it got added after https://github.com/pinterest/ktlint/pull/707
